spec/services/message_service_spec.rb

```diff
@@ -39,6 +41,9 @@
       before(:each){ service.create }
       subject{ service.resource }
       its(:user){ is_expected.to eql current_user }
+      it 'should change the conversations updated_at timestamp' do
+        expect { service.build }.to change { conversation.updated_at }
```

Comment:
After adding `reload`, it still fails
![screen shot 2017-05-18 at 11 27 07](https://cloud.githubusercontent.com/assets/8989606/26198253/4a640fec-3bbd-11e7-957f-60565e3c177c.png)